android/app/src/main/AndroidManifest.xml

```diff
@@ -48,7 +48,7 @@
         </provider>
         <activity
             android:name=".MainActivity"
-            android:launchMode="singleTop"
+            android:launchMode="singleTask"
```

Comment:
Why we switching here to singleTask it would work more like a singleton, explain pls

Reply:
I've thought it should be only one app instance in there. Because of when user is on instance A, then they click on link to open app (new instance B will be created). So, there will be 2 instances from Recent menu (Android OS), this may make user confuse.
But I decide to revert this into `singleTop` because two reasons:
- It may have some conflicts with another dependences/library settings in the project which may cause the application's current functions to stop working
- Some other apps (that have dynamic links feature) have multiple instance too